controllers/csiscaleoperator_controller.go

```diff
@@ -324,40 +338,48 @@ func (r *CSIScaleOperatorReconciler) getAccessorAndFinalizerName() (metav1.Objec
 	return accessor, finalizerName, nil
 }
 
-func (r *CSIScaleOperatorReconciler) deleteClusterRolesAndBindings() error {
-	/*	if err := r.deleteClusterRoleBindings(instance); err != nil {
+func (r *CSIScaleOperatorReconciler) deleteClusterRolesAndBindings(instance *csiscaleoperator.CSIScaleOperator) error {
+	logger := csiLog.WithName("in deleteClusterRolesAndBindings")
```

Comment:
Rather then passing `in deleteClusterRolesAndBindings` to WithName method, we can just pass the method name like `csiLog.WithName("deleteClusterRolesAndBindings".

This would look like this in operator log.
```log
<DATE> <LOGTYPE> csiscaleoperator.deleteClusterRolesAndBindings <LOGMESSAGE>
```
we can do same in all other places too.

Reply:
Right. Done. We will do at other places as part of clean up process.